github/search.go

```diff
@@ -54,6 +55,37 @@ func (s *SearchService) Repositories(query string, opt *SearchOptions) (*Reposit
 	return result, resp, err
 }
 
+// CommitsSearchResult represents the result of a commits search.
+type CommitsSearchResult struct {
+	Total             *int           `json:"total_count,omitempty"`
```

Comment:
Wouldn't it reduce confusion if the Go field name matched the JSON field name?
In other words, use `TotalCount` here?